src/panels/lovelace/create-element/create-row-element.ts

```diff
@@ -41,6 +41,7 @@ const LAZY_LOAD_TYPES = {
   cast: () => import("../special-rows/hui-cast-row"),
   buttons: () => import("../special-rows/hui-buttons-row"),
   attribute: () => import("../special-rows/hui-attribute-row"),
+  text: () => import("../special-rows/hui-text-row"),
```

Comment:
Why do we need to Lazy load?

Reply:
Lazy load is the default. Only ones that are super popular should be considered to be bundled. Requires an editor too.